simulator/infra/replacement/cache_replacement.cpp

```diff
@@ -89,7 +90,7 @@ class Pseudo_LRUCacheInfo : public CacheReplacementInterface
     public:
         explicit Pseudo_LRUCacheInfo( std::size_t ways);
         void touch( std::size_t way) override;
-        void set_to_erase( std::size_t ) override;
+        void set_to_erase( std::size_t /* unused */) override;
```

Comment:
I don't understand this commentary. The fact that there is no variable name indicates that the variable is unused, doesn't it?

Reply:
No, it does not indicate. The absence of a parameter may be a typo:

```c++
aufo func(uint )
{
// some long code
    unit parameter = // ...
// some long code
    call_another_func(parameter)
}
```

To distinguish between intended unused parameters and typos, Clang-Tidy suggests to be verbose and we follow this direction. Additionally, we may put the actual name here if we decide to use it.